azkaban-common/src/main/java/azkaban/project/ProjectManager.java

```diff
@@ -80,6 +80,7 @@ public ProjectManager(ProjectLoader loader, Props props) {
     // itself.
     Props prop = new Props(props);
     prop.put(ValidatorConfigs.PROJECT_ARCHIVE_FILE_PATH, "initialize");
+    new XmlValidatorManager(prop);
```

Comment:
Why do we need to create an XmlValidatorManager here even though we don't store it in any variable? It would be good to add a comment explaining this.